test-cases/cdc/cdc-15m-replication-gemini.yaml

```diff
@@ -23,6 +23,9 @@ nemesis_interval: 1
 extra_network_interface: true
 
 gemini_cmd: "gemini -d --duration 15m --warmup 0s -c 5 -m write --non-interactive --cql-features basic --max-mutation-retries 100 --max-mutation-retries-backoff 100ms --replication-strategy \"{'class': 'SimpleStrategy', 'replication_factor': '3'}\" --table-options \"cdc = {'enabled': true}\""
+# Note: for preimage and postimage we use 1 thread because there is no sensible way to test pre/post-images with concurrent writes happening to a single row.
+gemini_cmd_preimage: "gemini -d --duration 15m --warmup 0s -c 1 -m write --non-interactive --cql-features basic --max-mutation-retries 100 --max-mutation-retries-backoff 100ms --replication-strategy \"{'class': 'SimpleStrategy', 'replication_factor': '3'}\" --table-options \"cdc = {'enabled': true, 'preimage': true}\""
+gemini_cmd_postimage: "gemini -d --duration 15m --warmup 0s -c 1 -m write --non-interactive --cql-features basic --max-mutation-retries 100 --max-mutation-retries-backoff 100ms --replication-strategy \"{'class': 'SimpleStrategy', 'replication_factor': '3'}\" --table-options \"cdc = {'enabled': true, 'postimage': true}\""
```

Comment:
Test will faile, because we don't have new parameters 'gemini_cmd_preimage' and 'gemini_cmd_postimage' in sct_config.py defined.
First_way: Need to define new config parameters in sct_config.py
Another_way: just create new test config file with single gemini_cmd parameter and set it as test_config in pipeline files

Reply:
Fixed